bower.json

```diff
@@ -4,10 +4,13 @@
     "homepage": "https://github.com/aspnet/jquery-validation-unobtrusive",
     "description": "Add-on to jQuery Validation to enable unobtrusive validation options in data-* attributes.",
     "main": [
-        "jquery.validate.unobtrusive.js",
-        "jquery.validate.unobtrusive.min.js"
+        "jquery.validate.unobtrusive.js"
+    ],
+    "ignore": [
+        "*.json",
```

Comment:
should add `.gitignore`..no? BTW I noticed that even though we try to ignore `bower.json` here it still gets installed by bower when installing a package.

Thanks! for updating the other repo too